.github/workflows/tests.yaml

```diff
@@ -18,20 +18,28 @@ jobs:
       matrix:
         operating-system: [ ubuntu-latest ]
         php-version: [ '8.0', '8.1', '8.2' ]
+        # TODO: include highest & lowest
+        dependency: [ locked ]
+        # TODO: include phpdbg later
```

Comment:
btw, we don't need phpdbg as PHPUnit dropped support for it and we removed it https://github.com/infection/infection/pull/1363